Xamarin.Forms.Core/Visual.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace Xamarin.Forms
+{
+	public static class VisualMarker
+	{
+		public static IVisual MatchParent { get; } = new VisualRendererMarker.MatchParent();
+		public static IVisual Default { get; } = new VisualRendererMarker.Default();
+		public static IVisual Material { get; } = new VisualRendererMarker.Material();
+	}
+
+	public static class VisualRendererMarker
```

Comment:
I think this one should't be public. or if they are, the ctor for the nested classes should be internal to prevent instantiation